src/features/calendar/components/CalendarDayView/Day/index.tsx

```diff
@@ -27,9 +27,15 @@ const Day = ({ date, dayInfo }: { date: Date; dayInfo: DaySummary }) => {
         gap={1}
         justifyItems="flex-start"
       >
-        {dayInfo.events.map((event, index) => {
-          return <Event key={index} event={event.data} />;
```

Comment:
You probably already know this about react (and I know this problem was already there before but I'm pointing it out since it remains in your code), but using index as a `key` is a hack typically only employed when there is no better unique identifier for an element.

The purpose of the `key` property is to allow the React diffing algorithm (that compares the React elements returned by our components with the current state of the DOM) to figure out whether a certain element was an addition, removal or update of an existing element. It's especially important when looping over a list, and especially so if the order in that list might change. That way, React can move DOM elements around if events reorder, rather than updating the DOM elements (as if the individual events had changed).

Using array index as key basically provides no additional information and does not help React in understanding that the order changed.

In this case, I'm not sure why this code isn't already using `event.data.id` which is already unique. But again, that wasn't the purpose of this PR and the problem was already there, so I'm just going to approve as is and you can create a separate PR if you do want to make that change later. 😊